src/__experimental__/components/input/input.stories.js

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+  boolean, number, select
+} from '@storybook/addon-knobs';
+import { Store, State } from '@sambego/storybook-state';
+import { dlsThemeSelector, classicThemeSelector } from '../../../../.storybook/theme-selectors';
+import OptionsHelper from '../../../utils/helpers/options-helper';
+import info from './documentation/info';
+import { Checkbox } from '../checkbox';
+import Decimal from '../decimal';
+import GroupedCharacter from '../grouped-character';
+import NumberInput from '../number';
+import DateInput from '../date';
+import { RadioButtonGroup, RadioButton } from '../radio-button';
+import { Select, Option } from '../select';
+import Switch from '../switch';
+import Textarea from '../textarea';
+
+const store = new Store({
+  numberInputValue: '',
+  groupedCharacterValue: '',
+  checkboxValue: false,
+  dateValue: '2009-12-12',
+  radioButtonGroupValue: ''
```

Comment:
`radioButtonGroupValue` is not used anywhere :warning:

Reply:
removed

Reply:
Thanks :white_check_mark: